backend/tests/test_auth.py

```
@@ -3,23 +3,22 @@
 import pytest
 import uuid
 from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
 
 # Add backend directory to sys.path
 sys.path.append(str(Path(__file__).resolve().parents[1]))
 
 from main import app
-from database.session import SessionLocal, Base, engine
+from database.session import SessionLocal, engine
```

Ruff (0.8.2)

`database.session.engine` imported but unused

Remove unused import: `database.session.engine`

(F401)